### ISSUE TITLE
Interaction with TabMixPlus Session restore 

### ISSUE THREAD

**Issue by snowbound** [Quicksaver/Tab-Groups#36]
I use TabMixPlus extension a lot in Firefox 42 including its Session Restore option rather than using Firefox's built in restore. I find that TMP allows prompts me whether I want to save the session when I close the browser and when I reopen it. I came across your extension but notice that your extension is always warning me under "Keep Tab Groups across Restarts" even after I enable that option the next time I go into Tab Groups I see the yellow bar across the bottom of the screen.

The reason I like the ability to be able to not to restore tabs upon start up is that if I have an issue in Firefox and I need to test something I do not have to wait for the tabs to load. Only after fixing the issue will I then tell TMP to restore the prior session and its Tabs.

**Comment by Quicksaver** [comment]
Actually your groups should be restored if you choose to restore the session on startup, like TMP prompts you to. It just shows the warning because it doesn't know about this TMP feature. I'll change it so if you're using TMP's session restore, the warning won't be shown at all.

**Comment by snowbound** [comment]
Appears that there is also an issue with your extension and TMP 0.4.1.9 in that if your extension is enabled and TMP session manager is enabled I am always seeing the attached warning popping up when Firefox is started.
![capture](https://cloud.githubusercontent.com/assets/596147/11846630/16c3d6d0-a3df-11e5-8da1-b0a81abfd58d.PNG)

If I say no it still comes up. If I disable your extension in FFox 43 no more warning.

Is your extension saving Session only cookies and restoring them on the next start up of Firefox? I am asking because I use Gmail in tabs. In the past when I closed FFox and restarted I would need to login again which is what I want due to privacy/security. Ever since I had your extension installed I was always still logged into Gmail the next day when I started Firefox up.
